src/Backend/Modules/Authentication/Tests/Actions/IndexTest.php

```diff
@@ -77,5 +78,86 @@ public function testAuthenticationWithCorrectCredentials()
             'now editing:',
             $client->getResponse()->getContent()
         );
+
+        // logout to get rid of this session
+        $client->followRedirects(false);
+        $client->request('GET', '/private/en/authentication/logout');
+    }
+
+    public function testPagesUserWithCorrectCredentials()
+    {
+        // The authentication class persist the previous user.
+        // In practice this situation will almost never occur.
+        // Logging in one user only to log out and subsequently
+        // loggin in with another seems a bit much without
+        // a redirect in place to refresh the application.
+        // Another way to make this test, test. Would be to insulate
+        // the client in each test.
+        Authentication::tearDown();
```

Comment:
Can't you put this in a tearDown method in this class? This way, you don't need this duplicate 8 lines long comment (and you can add it as a method comment this way)